dev/Tiltfile

```diff
@@ -329,13 +329,13 @@ si_buck2_resource(
     resource_deps = [
         "postgres",
     ],
-    readiness_probe = probe(
-        period_secs = 5,
-        http_get = http_get_action(
-            port = 9001,
-            path = "/",
-        ),
-    ),
+    # readiness_probe = probe(
```

Comment:
This change is just because it's annoying seeing these pings in the logs. It only affects local dev and only for folks who are running local auth apis.